### ISSUE TITLE
Можно ли скомпилировать на другую архитектуру отличающийся от моей

### ISSUE THREAD

**Issue by Artem-coderh** [FWGS/hlsdk-portable#393]

**Comment by a1batross** [comment]
Да, почти всегда можно. https://ru.wikipedia.org/wiki/%D0%9A%D1%80%D0%BE%D1%81%D1%81-%D0%BA%D0%BE%D0%BC%D0%BF%D0%B8%D0%BB%D1%8F%D1%82%D0%BE%D1%80

Можно подробнее?

**Comment by Artem-coderh** [comment]
У меня ubuntu 22.04 на архитектуре amd64 я хотел скомпилировать на архитектуру armv7hf

**Comment by a1batross** [comment]
В репозиториях есть кросс компилятор под ARM. Просто подсунь его CMake через переменные окружения CC и CXX и всё заработает.

Учти, соберёт он под Linux.

**Comment by Artem-coderh** [comment]
Спасибо большое очень помогло
